package.json

```diff
@@ -189,7 +189,7 @@
     "lint": "yarn eslint tests/js src/sentry/static/sentry/app --ext .js,.jsx,.ts,.tsx",
     "lint:css": "yarn stylelint 'src/sentry/static/sentry/app/**/*.[jt]sx'",
     "dev": "(yarn check --verify-tree || yarn install --check-files) && sentry devserver",
-    "dev-server": "webpack-dev-server",
+    "dev-ui": "SENTRY_UI_DEV_ONLY=1 SENTRY_WEBPACK_PROXY_PORT=7999 yarn webpack-dev-server",
```

Comment:
Thinking we use a different port to not conflict with reverse proxy? Any suggestions?

Reply:
Yeah I think this is a good idea. Annoying to just randomly pick ports but 👍